astro/src/content/docs/customize/look-and-feel/kickstart-custom-theme.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Kickstart instance with custom theme
+title: Custom Themes And Kickstart
```

Comment:
Might make more sense to title it 'Theme Kickstarts' as the others.

Reply:
What does 'Theme Kickstarts' mean? 

The point of this article is that you can use kickstart to set up themes. How about `Setting Up A Theme with Kickstart`?